Makefile

```
@@ -1,25 +1,25 @@
 create-servers:
 	packer build packer/compute.packer.json
 
-init:
-	./create.sh $(cluster)
+create:
+	./create.sh $(workspace) $(workspaces)
 
 destroy:
-	./destroy.sh $(cluster)
+	./destroy.sh $(workspace) $(workspaces)
 
 patch:
 	@cd infra; \
-	./deploy.sh $(cluster);
+	./deploy.sh $(workspace) $(workspaces);
 
 setup:
 	@cd helm; \
-	/bin/bash deploy.sh $(cluster);
+	/bin/bash deploy.sh $(workspace) $(workspaces);
 	@cd scripts; \
-	/bin/bash ./create-cluster-defaults-secret.sh
+	/bin/bash ./create-workspace-defaults-secret.sh
 
 setup-orchestration:
 	@cd infra/orchestration; \
-	./deploy.sh $(cluster)
+	./deploy.sh $(workspace) $(workspaces)
```

```suggestion
	./deploy.sh $(workspace) $(path)
```

Workspaces isn't very clear to the user what they should be passing.